.packit.yaml

```
@@ -38,3 +38,20 @@ jobs:
       # do not get the version from a tag (git describe) but from the spec file
       get-current-version:
         - grep -oP '^Version:\s+\K\S+' packaging/command-line-assistant.spec
+  - job: tests
+    trigger: pull_request
+    fmf_url: "https://gitlab.cee.redhat.com/rhel-lightspeed/enhanced-shell/cla-tests"
```

```suggestion
  - job: tests
    manual_trigger: true
    merge_pr_in_ci: false
    trigger: pull_request
    fmf_url: "https://gitlab.cee.redhat.com/rhel-lightspeed/enhanced-shell/cla-tests"
```
I would consider using `manual_trigger: true` and `merge_pr_in_ci: false`.
I don't see setting the manual_trigger that necessary, since running in container might not burn as many resources as requesting and running an image.
We will most probably run just a very small sanity subset of tests here and a larger portion will be handled elsewhere, so we can keep it triggered by any push.
merge_pr_in_ci might be more relevant, but not that important in our case, I think.